humblebundle.py

```diff
@@ -728,10 +736,15 @@ def main(argv=None):
         hb.update()
 
     if args.list is not None:
+        if args.platform is not None:
+            keys = filterGames(hb.games,args.platform)
```

Comment:
Also, platform filter should come after regex filter, because regex will usually filter to very few games, most likely a single one.